sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ThroughputProperties.java

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.implementation.Offer;
+import com.azure.cosmos.implementation.OfferAutoscaleSettings;
+
+/**
+ * Represents throughput of the resources in the Azure Cosmos DB service.
+ */
+public class ThroughputProperties {
```

Comment:
please add code sample snippet.
similar to `AnalyticalStorageCodeSnippet` or `IndexPolicyCodeSnippet` as example.

Reply:
Added snippet